library/globals.inc.php

```diff
@@ -259,7 +259,7 @@
       xl('Online Support Link'),
       'text',                           // data type
       'http://open-emr.org/',
```

Comment:
to what?

      From: Tony McCormick <notifications@github.com>
 To: LibreEHR/LibreEHR <LibreEHR@noreply.github.com> 
Cc: Terry Hill <teryhill@yahoo.com>; Mention <mention@noreply.github.com>
 Sent: Friday, July 29, 2016 6:23 PM
 Subject: Re: [LibreEHR/LibreEHR] Fix globals.inc (#32)
   
In library/globals.inc.php:> @@ -259,7 +259,7 @@
>        xl('Online Support Link'),
>        'text',                           // data type
>        'http://open-emr.org/',
Need to actually change the URL ... before we merge this one—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.